stanford/playbooks/analytics-jenkins.yml

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/cluster.yml
+  vars:
+    CLUSTER_NAME: analytics-jenkins
+  when: redeploy | default(False) | bool
+- name: "Setup {{ COMMON_HOSTNAME }}"
+  hosts: "tag_Cluster_{{ CLUSTER_NAME }}:&tag_todo_deploy:&tag_Deployment_{{ COMMON_DEPLOYMENT }}:&tag_Number_{{ CLUSTER_NUMBER }}"
+  gather_facts: True
+  become: true
+
+- name: Deploy analytics jenkins service
+  hosts: "tag_Cluster_{{ CLUSTER_NAME }}:&tag_todo_deploy:&tag_Deployment_{{ COMMON_DEPLOYMENT }}:&tag_Number_{{ CLUSTER_NUMBER }}"
+  gather_facts: True
+  become: True
+  vars:
+    CLUSTER_NAME: analytics-jenkins
```

Comment:
We may want to consider choosing an alternative name for this.

Two issues I ran into with naming were:
- length restrictions
- character restrictions

One potential hiccup I ran into with other clusters was that the names of some resources (ELBs, security groups, etc.) had surprisingly short maximum lengths for naming. I can't remember which resources, but I definitely remember it happening with `elasticsearch-forum`. Some resources were fine, but then you'd hit some examples like `lagunita-group-elasticsearch-forum-elb-public` which would fail.
Again, I don't remember what all was affected, but it's something to consider.

The other issue was with special characters. It may have been hyphens and/or underscores (again, I admittedly don't remember, sorry), but that prevented some resources from being created.

Given that, I've been attempting to keep cluster names short and strictly using lowercase alpha characters.

So, not an absolute deal breaker, but if we can find an alternative, great. If not, it'll be something to keep an out for.

The first thing that jumps to mind is "scheduler", though maybe that's too ambiguous?
I'll brainstorm over the weekend.